qdb/compat.py

```diff
@@ -23,14 +38,18 @@
     from functools import reduce
     PY2 = False
 
-
 PY3 = not PY2
 
 try:
     import gevent
+
+    def gyield():
+        gevent.sleep(0.01)
```

Comment:
A lot of the tests work by pushing some data into a queue or socket, yielding so that the greenlet reading from that channel can process them, and then making assertions on those actions. When this was 0s, we would yield back to the first thread before the actions had been completed resulting in test hangs.